content/glint-1-0-released.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Announcing Glint 1.0
+authors:
+  - dan-freeman
+  - the-ember-typescript-team
+date: 2023-04-27T15:00:00.000Z
+tags:
+  - '2023'
+  - typescript
+  - announcement
+---
+
+After two years in public preview and contributions from dozens of community members, the Ember
+TypeScript team is excited to announce the release of [Glint 1.0][glint-repo]!
+
+While Glint has been available as an unstable prerelease for some time, the project has now
+reached a level of stability and functionality that we're comfortable recommending it even for
+large applications with a low tolerance for churn.
+
+## What is Glint?
+
+In an Ember application, templates are the connective glue that bind together all the different
+pieces of your codebase. On its own, however, TypeScript can't see what's happening in your
+templates, so any typechecking or other helpful aids it provides can only ever operate upon
+small islands of disconnected code, and all of your connections are completely opaque to the
+tooling.
+
+[Glint][glint-docs] enables TypeScript to understand your templates as well, meaning you get
+end-to-end type safety for your entire application. You also get editor support for things like
+documentation and type info on hover, jumping to definitions, and automated codefixes and
+refactorings.
+
+![A component argument is renamed in a TS file and propagated to templates that reference it, then vice versa.](/images/blog/2023-04-27-glint-1-0/rename.gif)
+
+<!-- alex ignore just -->
+
+And, just like your editor can use TypeScript under the covers to provide many of those features
+even in vanilla JavaScript files, Glint can do the same for templates with a JS backing class.
```

Comment:
A thought, on reading this: we should *show* the easy example of a backing class, but I think as written it slightly implies that it *only* works with backing classes. I assume you're going that way because it doesn't work with template-only components without some MASSIVE hoop jumping, but it'd be good not to imply it, I think?

```suggestion
even in vanilla JavaScript files, Glint can do the same! For example, here Glint is showing info in a template with a JS backing class:
```